src/components/content/order/create/OrderSubmit.tsx

```diff
@@ -125,6 +152,7 @@ function OrderSubmit(props: OrderSubmitProps): JSX.Element {
         createRequest.serviceRequestProperties = serviceRequestProperties;
         submitDeploymentRequest.mutate(createRequest);
         setIsShowDeploymentResult(true);
+        setParams(customerServiceName, props, parameters);
```

Comment:
not sure if this is already covered. We must save the values even when the order is not submitted. 
When we click on "back" without submitting the order, still the values must be saved.